### ISSUE TITLE
Received wrong message

### ISSUE THREAD

**Issue by Kpaubert** [AndreasWintherMoen/jobb#80]
Hey!

Yesterday at 11:55 I received a text message for registration to this event: https://online.ntnu.no/events/2042. Apparently the registration does not start before 12:00 today. Any idea how this could happen in the production environment?

Thanks!

**Comment by AndreasWintherMoen** [comment]
Yeah, me too. The way it works is I schedule SMS for the next 24 hours at 3:30am. What happened here is they changed the signup date in the morning right before the signup. It's possible to do a new check before actually sending the SMS though.

**Comment by Kpaubert** [comment]
Ah, that clears things up :) Thanks for the swift reply!

Feel free to close this if you feel it's unnecessary to make the change at this point.
